src/librustc_metadata/rmeta/table.rs

```diff
@@ -61,7 +58,7 @@ macro_rules! fixed_size_encoding_byte_len_and_defaults {
             };
             self.write_to_bytes(&mut b[i]);
         }
-    }
+    };
```

Comment:
Curious spurious (but harmless) changes?

Reply:
These are rustfmt-caused (from the update, I'm pretty sure, not my editor :)